docker/pdo_services_base.dockerfile

```diff
@@ -99,4 +146,7 @@ RUN useradd -m -u $UID -g $GID -d /project/pdo -o -s /bin/bash $UNAME
 RUN chown --recursive $UNAME:$UNAME /project/pdo
 USER $UNAME
 
+#this is necessary for git operations such as "git describe --tags" from the new user
+RUN git config --global --add safe.directory ${DCAP_PRIMITIVES}
+
```

Comment:
why do we need git describe? this seems to have nothing to do with dcap. if its really necessary, pull it out into its own PR.

Reply:
removed -- I switched completely to dcap 1.22 (it's not the case to support 1.19 at this point), so that's not necessary.